src/yamlfix/services.py

```diff
@@ -128,6 +141,23 @@ def fix_code(source_code: str) -> str:
     else:
         shebang = ""
 
+    configurers = [
```

Comment:
I'd extract all the configuration of the yaml formatter to an internal function to keep `fix_code` simpler to read

Reply:
Or maybe create a new class `Yaml` in `adapters.py` that acts as interface over ruyaml hiding the details to this service.